tsdoc/src/nodes/DocParamBlock.ts

```diff
@@ -15,14 +15,17 @@ export interface IDocParamBlockParameters extends IDocBlockParameters {
  */
 export interface IDocParamBlockParsedParameters extends IDocBlockParsedParameters {
   spacingBeforeParameterNameExcerpt?: TokenSequence;
-
   parameterNameExcerpt: TokenSequence;
   parameterName: string;
-
   spacingAfterParameterNameExcerpt?: TokenSequence;
-
-  hyphenExcerpt: TokenSequence;
+  hyphenExcerpt?: TokenSequence;
   spacingAfterHyphenExcerpt?: TokenSequence;
+
+  unsupportedJsdocTypeBeforeParameterNameExcerpt?: TokenSequence;
+  unsupportedJsdocOptionalNameOpenBracketExcerpt?: TokenSequence;
+  unsupportedJsdocOptionalNameRestExcerpt?: TokenSequence;
+  unsupportedJsdocTypeAfterParameterNameExcerpt?: TokenSequence;
+  unsupportedJsdocTypeAfterHyphenExcerpt?: TokenSequence;
```

Comment:
The excerpts should go in the order they appeared in the original input.

Reply:
I've amended the commit to reflect this.